tests/App/config.yml

```diff
@@ -18,3 +18,7 @@ services:
     Sonata\BlockBundle\Tests\App\Controller\DemoController:
         tags:
             - controller.service_arguments
+
+# NEXT_MAJOR: remove this block
+sonata_block:
+    http_cache: false
```

Comment:
so my idea for the upgrade path would be to encourage users to upgrade to this config to disabled http caching fully.

In 5.x we can then make this a no-op as it will not do anything anymore and also deprecate it. On 6.0 it can be removed then.

WDYT?

Reply:
I think we don't have a better way, so I'm good with this.